README.md

```diff
@@ -2,12 +2,46 @@
 
 # dotnet-jack
 
-*Jack's love of dice has brought him here, where he has taken on the form of a Haskell library, in order to help you gamble with your properties.*
+A modern property-based testing tool, in the spirit of John Hughes & Koen Classen's [QuickCheck](https://web.archive.org/web/20160319204559/http://www.cs.tufts.edu/~nr/cs257/archive/john-hughes/quick.pdf). The key improvement is that shrinking is baked into the `Gen` monad, so you get it for free.
 
-![](img/dice.jpg)
+![](https://github.com/moodmosaic/dotnet-jack/raw/master/img/dice.jpg)
 
-Jack is a modern testing library, in the spirit of John Hughes & Koen Classen's Haskell [QuickCheck](https://web.archive.org/web/20160319204559/http://www.cs.tufts.edu/~nr/cs257/archive/john-hughes/quick.pdf).
+*Jack's love of dice has brought him here, where he has taken on the form of an F# library, in order to help you gamble with your properties.*
 
-What makes Jack different from Haskell QuickCheck is that instead of generating a random value and using a shrinking function after the fact, Jack generates the random value and all the possible shrinks in a rose tree, all at once. Shrinking is baked in the `Gen` computation expression, so you get it for free.
+## Highlights
 
-There isn't much here yet, but Jack can be published on NuGet once it's in a usable state.
+* TODO
+* TODO
+* TODO
+
+## At a glance
+
+`dotnet-jack` is a lightweight, but powerful, property-based testing tool. The key improvement being that shrinking is baked in to the `Gen` monad, so you get it for free.
+
+### Getting Started
+
+TODO
+
+### Generators
+
+TODO
+
+### Properties
+
+TODO
+
+## Limitations
+
+TODO
+
+## NuGet
+
+There isn't much here yet, but `dotnet-jack` can be published on NuGet once it's in a usable state.
+
+## Versioning
+
+`dotnet-jack` follows [Semantic Versioning 2.0.0](http://semver.org/spec/v2.0.0.html) once it’s in a usable state.
```

Comment:
I've mentioned [SemVer](http://semver.org/spec/v2.0.0.html) here, but if you disagree I'll remove it :smiley: